src/common/dom/apply_themes_on_element.ts

```diff
@@ -97,10 +98,10 @@ export const applyThemesOnElement = (
     if (accentColor) {
       cacheKey = `${cacheKey}__accent_${accentColor}`;
       themeRules["accent-color"] = accentColor;
-      const rgbAccentColor = hex2rgb(accentColor);
-      themeRules["text-accent-color"] =
-        rgbContrast(rgbAccentColor, [33, 33, 33]) < 6 ? "#fff" : "#212121";
     }
+    const rgbAccentColor = hex2rgb(accentColor || DefaultAccentColor);
+    themeRules["text-accent-color"] =
+      rgbContrast(rgbAccentColor, [33, 33, 33]) < 6 ? "#fff" : "#212121";
```

Comment:
Not sure if I should have just hardcoded this for the default instead of doing the calculation, that might be more brittle though if the default ever changes.

Reply:
We should only apply this if the accent color is not the default, so keep it in the if like before. Otherwise developers will be confused when they change the default `text-accent-color` variable and it doesn't update because it is overridden here.

Reply:
Yeah, then I would define a default `text-accent-color`. I was assuming that would already exist, my bad.